components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/model/SAMLSSOServiceProviderDO.java

```diff
@@ -78,6 +81,8 @@ public class SAMLSSOServiceProviderDO implements Serializable {
     private boolean doFrontChannelLogout;
     private String frontChannelLogoutBinding;
 
+    private static final String BACKCHANNEL_LOGOUT_BINDING = "BackChannel";
```

Comment:
Better to move out this constant to a constants class.

Reply:
Resolved in [Introduce JDBC based persistence for SAML](https://github.com/wso2/carbon-identity-framework/pull/6241/commits/d1e67ce649f7a5d2e7e62f364ce93815f347e7b4)